src/main/java/com/ciandt/techgallery/persistence/model/TechnologyFollowers.java

```diff
@@ -0,0 +1,61 @@
+package com.ciandt.techgallery.persistence.model;
+
+import com.googlecode.objectify.Ref;
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.annotation.Unindex;
+
+import java.util.List;
+
+/**
+ * TechnologyFolowers entity.
+ * 
+ * @author ibrahim
+ *
+ */
+@Entity
+public class TechnologyFollowers extends BaseEntity<Long> {
+
+  /*
+   * Constants --------------------------------------------
+   */
+  public static final String ID = "Id";
+  public static final String TECHNOLOGY = "technology";
+  public static final String FOLLOWERS = "followers";
+
+  @Id
+  Long id;
```

Comment:
Please use the same id as Technology. Use a String id and fill it with the technology id.
When using transactions is preferred to have a known id.